docker-compose.yml

```diff
@@ -10,7 +10,7 @@ sut:
     - bitcoind-username-only
 
 bitcoind:
-  image: seegno/bitcoind:0.13-alpine
+  image: seegno/bitcoind:0.13.2-alpine
```

Comment:
I did not add support for this bump.
I agree that we should pin to older version and create a new PR for bumping to 0.13.2.